server/src/domain/user/response-dto/user-available-version.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AvailableVersionResponseDto {
```

Comment:
This data doesn't really have anything to do with the user resource, so I find it a bit weird to return it here. We already have `GET /server-info/version`, so maybe something like `GET /server-info/available-version` would make more sense.